sync.js

```diff
@@ -242,16 +249,20 @@ class Sync extends events.EventEmitter {
           }
         })
         pump(stream, connection, stream, function (err) {
+          if (--self._activeSyncs === 0) {
+            self.osm.core.resume()
+          }
           if (peer.sync) {
             if (stream.goodFinish) {
               peer.status = 'replication-complete'
               peer.message = undefined
-              return peer.sync.emit('end')
+              peer.sync.emit('end')
```

Comment:
I think we could do without these changes here from 259-263, it shouldn't change things and i believe this is the cause of the merge conflict

Reply:
@noffle  i rebased your pause-resume on top of my sync-manager branch and resolved the conflict